controllers/RemindersController.js

```diff
@@ -0,0 +1,138 @@
+        }
+};
+
+const getHealthReminderOptions = async ( ack, body, client, logger, say ) => {
+        try {
+                // send acknowledgement back to slack over the socketMode websocket connection
+                // this is so slack knows you have received the event and are processing it
+                const fullPath = path.resolve("resources/HealthReminders.json");
+                const healthRemsView = await fs.readFileSync(fullPath);
+                await say(JSON.parse(healthRemsView));
+
+        } catch (error) {
+                console.log('An error occurred', error);
+        }
+};
+
+const setReminders = async ({ack, body, client, logger, say }) => {
+    try{
+        await ack();
+
+        const remType = body.actions[0].selected_option.value;
+        console.log(remType);
+        if(remType == 'add-reminder'){
+            getReminderOptions(ack, body, client, logger, say);
+        }else if(remType == 'health-reminders'){
+            getHealthReminderOptions(ack, body, client, logger, say);
+        }
+    }catch (error) {
+        console.log('An error occurred', error);
+    }
+};
+
+const addHealthReminder = async ({ack, body, client, logger, say }) => {
+    const { SocketModeClient } = require('@slack/socket-mode');
+    const { WebClient } = require('@slack/web-api');
+
+    const appToken = process.env.APP_TOKEN;
+    const socketModeClient = new SocketModeClient({ appToken });
+    const userToken = process.env.USER_OAUTH_TOKEN;
+
+    const webClient = new WebClient(process.env.SLACK_BOT_TOKEN);
+    try {
+        await ack();
+        console.log(JSON.stringify(body));
+        const message = body.state.values['health-reminder-radio-options']['radio_buttons-action']['selected_option'].value;
+        console.log(JSON.stringify(message));
+        const time = body.state.values['health-reminder-time-dropDown']['reminder-health-time']['selected_option'].value;
+        console.log(JSON.stringify(time));
+        await webClient.reminders.add({
+                                token:userToken,
+                                text:message,
+                                time:time
+        });
+        const fullPath = path.resolve("resources/ReminderCreatedView.json");
+        const remCreatedView = await fs.readFileSync(fullPath);
+        await say(JSON.parse(remCreatedView));
+    }catch (error) {
+        console.log('An error occurred', error);
+    }
+};
+
+const addReminder = async ({ack, body, client, logger, say }) => {
+    const { SocketModeClient } = require('@slack/socket-mode');
+    const { WebClient } = require('@slack/web-api');
+
+    const appToken = process.env.APP_TOKEN;
+    const socketModeClient = new SocketModeClient({ appToken });
+    const userToken = process.env.USER_OAUTH_TOKEN;
+
+    const webClient = new WebClient(process.env.SLACK_BOT_TOKEN);
+    try {
+        await ack();
+        console.log(JSON.stringify(body));
```

Comment:
@anwebanSalesforce please remove console logs, unnecessarily it will add overheads.

Reply:
@chandel2212 Done. Please re-review.